packages/insights/src/db/schema.ts

```diff
@@ -194,3 +194,24 @@ export const edgeTable = sqliteTable(
     ),
   })
 );
+
+export const routesTable = sqliteTable(
+  'routes',
+  {
+    id: integer('id').primaryKey(),
+    publicApiKey: text('public_api_key').references(() => applicationTable.publicApiKey),
+    manifestHash: text('manifest_hash').notNull(),
+    route: text('route').notNull(),
+    symbol: text('symbol').notNull(),
+    interactions: integer('interactions').notNull(),
+    timeline: integer('timeline').notNull(),
```

Comment:
This `timeline` needs to be expanded to `timeline00`, ... `timeline49`

Reply:
Fixed [2c1794e](https://github.com/BuilderIO/qwik/pull/4971/commits/2c1794e964d4a0127e88140005e89b3293e2f161)